extensions/amp-lightbox-gallery/amp-lightbox-gallery.md

```diff
@@ -116,16 +116,14 @@ Lightboxed items have a thumbnail gallery view. You can optionally specify a thu
 #### Example: using `lightbox-thumbnail-id` to specify a thumbnail
```

Comment:
/cc @bpaduch replaced the `<amp-facebook>` with `<amp-youtube>` instead. There's a tested version of this that works in the `test/manual/amp-lightbox-gallery-launch.amp.html`.